### PR TITLE
Added Hindi Language file .

### PR DESCRIPTION
**Pull request by vaibhavbansal23**

### CODE REVIEW

File: lang.hi.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+------------------*/
+
+$lang = array();
+
+$lang['BEEIMG'] = 'BeeIMG';
+
+// Titles
+$lang['TITLE_HOME'] = 'मुफ्त छवि होस्टिंग और साझा';
+$lang['TITLE_UPLOAD'] = '
+छवियों को अपलोड करें';
```
Comment (miyurusankalpa) [inline lang.hi.php]: no new lines, please

File: lang.hi.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+$lang['MENU_STATS_TITLE'] = '
+हमारे आँकड़े देखें';
+$lang['MENU_TOP'] = 'शीर्ष 9';
+$lang['MENU_TOP_TITLE'] = 'सारे टॉप 9 छवियों';
+$lang['MENU_FOLLOW'] = 'का पालन करें';
+
+// Upload Carousel
+$lang['UPLOAD_CAR_LP'] = 'घर';
+$lang['UPLOAD_CAR_1'] = '
+कोई आईपी \ 's अपलोड पर दर्ज किया जाएगा।';
```
Comment (miyurusankalpa) [inline lang.hi.php]: add a slash before the quote, no spaces, build fails because of this

**Comment by miyurusankalpa** [comment]
can you fix those? the build fails on line 47
thanks btw

**Comment by vaibhavbansal23** [comment]
On it

**Comment by miyurusankalpa** [comment]
merged, thanks 👍
have a great hacktoberfest 😃
*Good Morning btw :D*

**Pull request merged**